packages/ui/Scrollable/Scrollable.js

```diff
@@ -598,21 +794,24 @@ const ScrollableBase = forwardRef((props, reference) => {
 		return Math.min(1, 2 * overDistance / baseSize);
 	}
 
-	function applyOverscrollEffect (orientation, edge, type, ratio) {
-		props.applyOverscrollEffect(orientation, edge, type, ratio);
-		setOverscrollStatus(orientation, edge, type === overscrollTypeOnce ? overscrollTypeDone : type, ratio);
+	function applyOverscrollEffect (orientation, edge, overScrollType, ratio) {
+		props.applyOverscrollEffect(orientation, edge, overScrollType, ratio);
+		setOverscrollStatus(orientation, edge, overScrollType === overscrollTypeOnce ? overscrollTypeDone : type, ratio);
 	}
 
-	function checkAndApplyOverscrollEffect (orientation, edge, type, ratio = 1) {
+	function checkAndApplyOverscrollEffect (orientation, edge, overScrollType, ratio = 1) {
 		const
 			isVertical = (orientation === 'vertical'),
 			curPos = isVertical ? variables.current.scrollTop : variables.current.scrollLeft,
 			maxPos = getScrollBounds()[isVertical ? 'maxTop' : 'maxLeft'];
 
-		if ((edge === 'before' && curPos <= 0) || (edge === 'after' && curPos >= maxPos)) { // Already on the edge
-			applyOverscrollEffect(orientation, edge, type, ratio);
+		if (
+			overScrollType === 'JS' && (edge === 'before' && curPos <= 0) || (edge === 'after' && curPos >= maxPos) ||
```

Comment:
why it should be changed?
overscroll effect is not working due to this.

type have "JS" or "Native" but there is "2" value in overScrollType